### ISSUE TITLE
Just some findings with 0.14.12

### ISSUE THREAD

**Issue by Timo-Fi** [forrestguice/SuntimesWidget#677]
Hello again, to recap I use the 'Voice Notify' app, it speaks out the notifications (and alarms) that go to the Notification Shade, I have allowed speaking only for the SunTimes app.  

With 0.14.12 it now happens that whenever I clear the 'Notification Shade', I hear "Alarm" in case only one Suntimes message is cleared,  otherwise "Alarms".  This is not an issue, they can be easily silenced in the 'Voice Notify' by 'TTS Text Replacement" option, by specifying both "Alarm ." and "alarms ." to be converted to "" (peculiarly there has to be a space before the dot, else I hear "done".  This is so in Android OS v.7, v.12 and v.13. Again, this is not an issue, I just write this in case there happens to be another soul on the globe who does the same as I, it was a little tricky task to silence the "done".  I'm still hoping really hard that the notifications could someday vanish automatically, without any user intervention.

Thank you very much the new substitution %eot (equation of time), only today I took it into use.  At the time when it was introduced I immediately went try it, but as I failed to find the description for it in the "?" I didn't follow through the testing.  Oh, and now, I'm looking for the value of the Sun Declination Angle ...

**Comment by forrestguice** [comment]
> whenever I clear the 'Notification Shade', I hear "Alarm" in case only one Suntimes message is cleared, otherwise "Alarms". This is not an issue, they can be easily silenced

I'm glad there is a workaround. It seems this is a side effect to some of the recent bug fixes (an additional notification is now shown for a split second in some circumstances). It definitely isn't intentional.

> I'm still hoping really hard that the notifications could someday vanish automatically, without any user intervention.

Yeah, I intended for `v0.15.0` to be released by now. I have actually been drafting [release notes](https://github.com/forrestguice/SuntimesWidget/pull/678) these past couple weeks, and trying to find bugs (which are numerous). I have to tell myself "perfect is the enemy of good" - the release should be sometime this week.

I'll add declination angle in `v0.15.1`. The "online help" ([wiki](https://github.com/forrestguice/SuntimesWidget/wiki/Help#data-substitutions)) should list the newer entries, but the in-app help generally lags behind (since updating it means synchronizing/updating many translations).

**Comment by Timo-Fi** [comment]
Hooray! An Android OS v.13 device with Suntimes 0.14.12 was left unattended over the weekend and today it was still happily talking. At first I couldn't believe my ears. There was only nine notifications in the  'Notification Shade' even if as many as about 60 notifications had been processed. So they do vanish properly now. I'm hoping to hear the same tomorrow with my OS versions 7 and 12 devices.  This is so very great, thank you very much!  Also thank you very much for approving the addition of the 'declination angle' data substitution,  while the value can be seen from the Sun Position Widget (3x2 Basic) approximately, accurate value is always more precise.

**Comment by forrestguice** [comment]
I wish I could take credit for the change in notification behavior in `v0.14.12` 😕. I suppose it might be an Android 13 feature (or maybe just a side-effect of recent changes).

At any rate, `v0.15.0` is now released - it will probably be available on F-Droid by the end of the week. I think its kind of rough (`.0` releases always are), but we can fix/refine the new features in later patches.

This release adds the "quick notification" type (it will be necessary to edit each notification and change its type). These should automatically dismiss after 30 seconds (configurable from alarm options). There are a few other features to look out for. The Sun Position dialog adds an "Altitude graph", with controls for seeking to a given angle. Support for custom altitude events has also been added (configure these from Settings -> User Interface).

> thank you very much!

You are very welcome. Thank you! 
I really appreciate all of the testing and thoughtful feedback. Also, I checked PayPal donations recently (something I do infrequently for mental health reasons), and noticed your very generous donation. It feels really good knowing the software is useful. Thank you. :)
